libs/storages/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@wallet/source",
+  "version": "0.2.0",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest",
+    "build": "tsc -p tsconfig.lib.json"
+  },
+  "private": true,
+  "dependencies": {
+    "@swc/helpers": "~0.5.2",
+    "idb": "^8.0.0"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.14.5",
+    "@babel/core": "^7.14.5",
+    "@swc/cli": "~0.1.62",
```

Comment:
I think it is missing typescript, maybe it needs to run:

npm install typescript --save-dev